pkg/action/env.container.go

```diff
@@ -29,7 +29,24 @@ type containerEnv struct {
 	driver driver.ContainerRunner
 	imgres ChainImageBuildResolver
 	dtype  driver.Type
-	prefix string
+	nprv   ContainerNameProvider
```

Comment:
I know the naming in go is shit and very opinionated, but it must be readable and easyly guessed.  
There is something I don't like about `nprv`, maybe it's better to use `nameprv`?